cli/install/install.go

```diff
@@ -495,7 +499,12 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 				return err
 			}
 			for _, ver := range versions {
-				if ttVersion == ver.Str || (ttVersion[:1] != "v" && "v"+ttVersion == ver.Str) {
+				versionMatches, err := regexp.Match(majorMinorPatchRegexp, []byte(ttVersion))
+				if err != nil {
+					return err
+				}
+				if ttVersion == ver.Str || (versionMatches && ttVersion[:1] != "v" &&
```

Comment:
The regular expression `^[0-9]+\.[0-9]+\.[0-9]+`
) excludes the case because it only matches `begin_of_a_string+digits.digits.digits`:
```suggestion
				if ttVersion == ver.Str || (versionMatches &&
```

Reply:
Deleted redundant expression.